commands/user/events/events.go

```diff
@@ -0,0 +1,73 @@
+package events
+
+import (
+	"fmt"
+
+	"github.com/profclems/glab/commands/cmdutils"
+	"github.com/profclems/glab/internal/utils"
+	"github.com/profclems/glab/pkg/api"
+	"github.com/profclems/glab/pkg/tableprinter"
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+)
+
+func NewCmdEvents(f *cmdutils.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "events",
+		Short: "View user events",
+		Args:  cobra.MaximumNArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			apiClient, err := f.HttpClient()
+			if err != nil {
+				return err
+			}
+
+			events, err := api.CurrentUserEvents(apiClient)
+			if err != nil {
+				return err
+			}
+			projects := make(map[int]*gitlab.Project)
+			for _, e := range events {
+				project, err := api.GetProject(apiClient, e.ProjectID)
+				if err != nil {
+					return err
+				}
+				projects[e.ProjectID] = project
+			}
+
+			title := utils.NewListTitle("User events")
+			title.CurrentPageTotal = len(events)
+
+			if err = f.IO.StartPager(); err != nil {
+				return err
+			}
+			defer f.IO.StopPager()
+			fmt.Fprintf(f.IO.StdOut, "%s\n%s\n", title.Describe(), DisplayAllEvents(events, projects))
+
+			return nil
+		},
+	}
+
+	return cmd
```

Comment:
Since the events endpoint is paginated, you should add `per-page` and `page` flags
```go
cmd.Flags().IntP("page", "p", 1, "Page number")
cmd.Flags().IntP("per-page", "P", 30, "Number of items to list per page. (default 30)")
```

Reply:
I think I read from the wrong place. The event API does not seem to be paginated but it has `before` and `after` params instead